projects/packages/sync/src/class-actions.php

```diff
@@ -194,6 +194,13 @@ public static function should_initialize_sender() {
 			return self::sync_via_cron_allowed();
 		}
 
+		/**
+		 * For now, if the feature flag is enabled we will always initialize send, even for GET and unauthenticated requests.
+		 */
+		if ( Settings::get_setting( 'dedicated_request_enable' ) ) {
```

Comment:
Wouldn't it be better to name this as `enabled`, instead of `enable`? 

Makes it more in-line with functions that will check for it `is_enabled` for example.

Reply:
Also we might want to extract this check to a method, so we don't have to check the setting every time.

Reply:
Great points! Do you think it makes more sense to rename this to something like `sync_spawning_enabled`?